composer.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "spryker-eco/unzer-api",
+    "type": "library",
+    "description": "UnzerApi module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.4",
+        "psr/http-message": "^1.0.1",
+        "spryker/guzzle": "^2.2.0",
+        "spryker/kernel": "^3.30.0",
+        "spryker/symfony": "^3.7.0",
```

Comment:
I don't see usage of this package

Reply:
it use `symfony/http-foundation`
in `SprykerEco\Zed\UnzerApi\Business\Api\Request` namespace 
it use `Symfony\Component\HttpFoundation\Request` at most of requests